### PR TITLE
Extensions module. Step 7

### PR DESCRIPTION
**Pull request by igorkorsukov**

### CODE REVIEW

File: src/framework/extensions/qml/Muse/Extensions/ExtensionsListPanel.qml [hunk 1]
```diff
@@ -190,12 +190,12 @@ Item {
             {"title": qsTrc("plugins", "Shortcut:"), "value": Boolean(selectedPlugin) ? selectedPlugin.shortcuts : ""}
```
Comment (Eism) [inline src/framework/extensions/qml/Muse/Extensions/ExtensionsListPanel.qml:190]: the context also should be `extensions`

File: src/framework/extensions/qml/Muse/Extensions/ExtensionsListPanel.qml [hunk 2]
```diff
@@ -152,7 +152,7 @@ Item {
                 selectedCategory: root.selectedCategory
                 selectedPluginCodeKey: prv.selectedPlugin ? prv.selectedPlugin.codeKey : ""
 
-                model: pluginsModel
+                model:  extensionsModel
```
Comment (Eism) [inline src/framework/extensions/qml/Muse/Extensions/ExtensionsListPanel.qml:155]: extra space

File: src/framework/extensions/internal/extensionsuiactions.cpp [hunk 3]
```diff
@@ -38,11 +34,11 @@ static UiAction MANAGE_ACTION = UiAction(
     "manage-plugins",
     mu::context::UiCtxAny,
     mu::context::CTX_ANY,
-    TranslatableString("action", "&Manage plugins…"),
-    TranslatableString("action", "Manage plugins…")
+    mu::TranslatableString("action", "&Manage plugins…"),
+    mu::TranslatableString("action", "Manage plugins…")
```
Comment (Eism) [inline src/framework/extensions/internal/extensionsuiactions.cpp:38]: should we rename it to Manage extensions?

File: src/framework/extensions/qml/Muse/Extensions/ExtensionsListPanel.qml (hunk 1 above)
Comment (igorkorsukov) [reply src/framework/extensions/qml/Muse/Extensions/ExtensionsListPanel.qml]: fixed

File: src/framework/extensions/qml/Muse/Extensions/ExtensionsListPanel.qml (hunk 2 above)
Comment (igorkorsukov) [reply src/framework/extensions/qml/Muse/Extensions/ExtensionsListPanel.qml]: fixed

File: src/framework/extensions/internal/extensionsuiactions.cpp (hunk 3 above)
Comment (igorkorsukov) [reply src/framework/extensions/internal/extensionsuiactions.cpp:38]: I asked Bradley..

**Review by Eism — APPROVED** [review]

**Pull request merged**